pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -947,6 +949,10 @@ private int addAndGetUnAckedMsgs(Consumer consumer, int ackedMessages) {
             subscription.addUnAckedMessages(ackedMessages);
             unackedMsgs = UNACKED_MESSAGES_UPDATER.addAndGet(consumer, ackedMessages);
         }
+        if (unackedMsgs < 0 && System.currentTimeMillis() - negtiveUnackedMsgsTimestamp >= 10_000) {
```

Comment:
@Technoboy- can you please provide motivation for this new logic in your PR description? This was not in the removed conditional here: https://github.com/apache/pulsar/pull/14246/files

Reply:
yes, in #14246, if there is a bug, the unackedMsgs is negative, there will print a lot of logs, this may confuse users.
So it's better to print at a period of 10s.